src/main/java/org/broadinstitute/barclay/argparser/CommandLineProgramProperties.java

```diff
@@ -27,5 +26,6 @@
      */
     String usageExample() default "The author of this program hasn't included any example usage, please complain to them.";
```

Comment:
@vdauwera I'm glad you caught this. This came from GATK4 (and there are only a handful of tools for which is populated), but it was never integrated with the CLP, and I didn't integrate it with docgen since it wasn't present in the GATK3 code base. So we need to figure out how/where it should surface. Again I think I'll make a separate ticket for that so as not to hold up the current PRs.

Also, BTW, Picard doesn't currently have this either; it looks like the usage summary there is included by convention in the "summary" output. It would be nice if they were separated since those strings are currently Picard-specific (i.e., they have embedded "java -jar picard.jar" references that won't make sense if displayed on the context of GATK).